example_data/epilepsy_bundle.json

```diff
@@ -0,0 +1,656 @@
+{
+  "resourceType": "Bundle",
+  "type": "transaction",
+  "entry": [
+    {
+      "fullUrl": "DiagnosticReport/43a930ca-d757-11ec-9d64-0242ac120002",
+      "resource": {
+        "resourceType": "DiagnosticReport",
+        "id": "43a930ca-d757-11ec-9d64-0242ac120002",
+        "status": "final",
+        "code": {
+          "coding": [
+            {
+              "system": "http://loinc.org",
+              "code": "81247-7",
+              "display": "Early onset or syndromic epilepsy"
+            }
+          ]
+        },
+        "subject": {
+          "type": "Patient",
+          "reference": "Patient/43a92b3e-d757-11ec-9d64-0242ac120002"
+        },
+        "performer": [
+          {
+            "type": "Practitioner",
+            "reference": "Practitioner/43a9517c-d757-11ec-9d64-0242ac120002"
+          }
+        ],
+        "specimen": [
+          {
+            "type": "Specimen",
+            "reference": "Specimen/43a96978-d757-11ec-9d64-0242ac120002"
+          }
+        ],
+        "conclusion": "Confirms the diagnosis of Childhood-nset epileptic encephalopathy (EEOC)."
+      },
+      "request": {
+        "method": "POST",
+        "url": "DiagnosticReport"
+      }
+    },
+    {
+      "fullUrl": "Observation/43a9579e-d757-11ec-9d64-0242ac120002",
+      "resource": {
+        "resourceType": "Observation",
+        "id": "43a9579e-d757-11ec-9d64-0242ac120002",
+        "status": "final",
+        "code": {
+          "coding": [
+            {
+              "system": "http://loinc.org",
+              "code": "69548-6",
+              "display": "Genetic variant accesment"
+            }
+          ]
+        },
+        "meta": {
+          "profile": [
+            "http://hl7.org/fhir/uv/genomic-reporting/gene-panel"
+          ]
+        },
+        "subject": {
+          "type": "Patient",
+          "reference": "Patient/43a92b3e-d757-11ec-9d64-0242ac120002"
+        },
+        "specimen": {
+          "type": "Specimen",
+          "reference": "Specimen/43a96978-d757-11ec-9d64-0242ac120002"
+        },
+        "performer": [
+          {
+            "type": "Practitioner",
+            "reference": "Practitioner/43a9517c-d757-11ec-9d64-0242ac120002"
+          }
+        ],
+        "component": [
+          {
+            "code": {
+              "coding": [
+                {
+                  "system": "http://loinc.org",
+                  "code": "48013-7",
+                  "display": "HGVS Description"
+                }
+              ]
+            },
+            "valueCodeableConcept": {
+              "coding": [
+                {
+                  "system": "http://www.ncbi.nlm.nih",
+                  "code": "gene_panel",
+                  "display": "NM_001271.3: c.3917del p.(Gln1306Argfs*13)"
+                }
```

Comment:
This is based on requirements and engagements with the Lab. Neil wanted us to report exactly what was on the PDF report. I have to formulate the bundle as you've pointed out using the codes. I searched the hgvs web ontologies for description at http://www.ncbi.nlm.nih and couldn't get anything. hence why I used the code gene_panel.

Reply:
I think that's my concern - as far as I can tell this isn't a coding system. Also a gene panel is a selection of genes to be sequenced, whereas this is the HGVS notation for the change. I imagine we should be aiming to use pre-existing coding systems, otherwise we should use free text?